### PR TITLE
Transparent menus

### PR DESCRIPTION
**Pull request by mohamedation**
this is what I meant. I changed the menu.svg opacity to 90% which I think makes the theme a little bit modern..for my taste at least.
screenshot from 2018-02-12 23-46-45-1

I hope you find that acceptable.

### CODE REVIEW

**Comment by fossfreedom** [comment]
Looks good to me - but I'm not a GNOME-Shell user.  @NicoHood Thoughts?

File: common/gnome-shell/3.18/dark-assets/menu/menu.svg [hunk 1]
```diff
@@ -14,8 +14,8 @@
    viewBox="0 0 32 32"
    id="svg5386"
    version="1.1"
-   inkscape:version="0.91 r13725"
-   sodipodi:docname="menu.svg">
+   inkscape:version="0.92.2 5c3e80d, 2017-08-06"
+   sodipodi:docname="menu (copy).svg">
```
Comment (NicoHood) [inline common/gnome-shell/3.18/dark-assets/menu/menu.svg]: Do we really want this name here?

**Comment by NicoHood** [comment]
Beside the content of the PR (the name "menu (copy).svg") I think it is very useful and looking good so far. But we need to make sure that the solid variant is non transparent.

**Comment by mohamedation** [comment]
how should we go about the solid variant? I think giving the overall theme style, the 90% opacity should work well without the need for a solid variant, but that's just me.

**Comment by NicoHood** [comment]
The arc theme is available in a complete solid variant for users who do not want transparency at all. We also need to add support for the solid theme. I currently aint got time to look into this. Maybe @fossfreedom  or you @mohamedation can figure it out :)

**Comment by mohamedation** [comment]
I will definitely give it a try :)
Thank you @NicoHood

**Comment by NicoHood** [comment]
@mohamedation any updates?

**Comment by mohamedation** [comment]
@NicoHood sadly, no. I don't really have a lot of free time lately and i really have no idea where to begin. I will try to make some time to look into it.

**Comment by FrancescoCuiusIuculano** [comment]
Hello, I have managed to solve the issue with the solid variant. However, I don't know how to add my commits to this PR. Should I create a new one, referencing it here?

**Comment by shoeper** [comment]
@FrancescoCuiusIuculano I think that would be the easiest thing to do.

**Comment by NicoHood** [comment]
@fossfreedom Why did you merge this? It was not ready yet and superceeded by @FrancescoCuiusIuculano PR

**Comment by fossfreedom** [comment]
Hmm... I tested this and it worked just fine.

If this was superseded then the PR should have been closed to prevent such confusion.

Will push a revert commit tonight.

So is the other PR ready for testing then? If it is I will run a few tests before merging.

**Comment by NicoHood** [comment]
This PR introduces transparency, even for the solid version of the theme.

The new PR tries to fix that, but I had no time to review it yet. It should be reviewed first.

Please revert the commit.

**Pull request merged**